docs/modules/commons/pages/expressions.adoc

```diff
@@ -40,6 +40,88 @@ Generates a random integer value between the specified origin (`minInclusive`) a
 |`1`
 |===
 
+=== `round`
+
+Rounds a number with an approximate value based on specific rounding definitions.
+
+[source, subs="+quotes"]
+----
+#{round(*$value*, *$maxFractionDigits*, *$roundingMode*)}
+----
+
+* *`$value`* - any number to be processed
+* *`$maxFractionDigits` (Optional)* - maximum number of digits after the decimal separator
+* *`$roundingMode` (Optional)* - specifies a rounding behavior for numerical operations (more information https://docs.oracle.com/javase/7/docs/api/java/math/RoundingMode.html[here]). 
+
+[cols="1, 1, 4", options="header"]
```

Comment:
this section should stick to roundingMode`, in other words the table should be a part of list item

Reply:
done